### DISCUSSION TITLE
Fuerzas en el modelo

### DISCUSSION THREAD

**Discussion by RobeReyes** [will-henney/globule-seminario#46] [General]
Comparación de la fuerza gravitacional por parte de la estrella con la fuerza de radiación

![IMG_20240119_091232](https://github.com/will-henney/globule-seminario/assets/123852273/6d2a1732-49aa-479e-ae06-b0d3db6c44b5)
Para el caso de la $v_{esc}=\sqrt{\frac{2GM}{r}}$ tenemos que por muy masiva la estrella, es de apenas 1 km/s

**Comment by will-henney** [comment]
![6d2a1732-49aa-479e-ae06-b0d3db6c44b5](https://github.com/will-henney/globule-seminario/assets/161264/8a75db3f-9b29-479b-85ab-3a6f4a3bb07a)

Here are some corrections to the radiation pressure ratio

**Comment by will-henney** [comment]
We need a more quantitative estimate of the stellar gravitational escape velocity. We can then compare this with the escape velocity from the surface of the globule due to its own mass

**Comment by will-henney** [comment]
Then we need to quantitatively compare the acceleration from gravity and radiation, which depends on the Eddington ratio multiplied by the ratio of cross sections. 

We will consider two different sorts of interaction between radiation and the matter in the photoevaporation flow:

1. Photoionization of neutral hydrogen
2. Absorption by dust grains that are collisionally coupled to the gas

**Comment by will-henney** [comment]
In both cases we need the cross section per hydrogen nucleon

1. For photoionization, this is the cross section $\sigma_0 (\nu/\nu_0)^{-3}$, averaged over the stellar spectrum (which yields $\approx 0.5 \sigma_0 \approx 3 \times 10^{-18}$, then multiplied by the neutral fraction $(1-x)$
2. For the dust, we can assume a standard cross section of $10^{-21}$ per H nucleon

**Comment by will-henney** [comment]
To calculate the ionization fraction $x$ we can assume photoionization equilibrium so long as the recombination time is short compared with the dynamic crossing time of the photoevaporation flow.

**Comment by will-henney** [comment]
Finally, the radiative acceleration need to be compared with the acceleration due to pressure gradients in the photoionized flow.  Hopefully it will turn out to be smaller, so that we do not need to modify the model.
